kotlin/.gitattributes

```diff
@@ -0,0 +1,31 @@
+# see https://help.github.com/articles/dealing-with-line-endings
```

Comment:
This is my default `.gitattributes` file. Shall we include it or leave it out, as the top level folder does not contain a gitattributes file?

Reply:
probably should have one at the top level actually